etc/Dictionaries/DFLError_en_GB.dic

```diff
@@ -29,6 +29,7 @@ UnknownProperty               =     couldn't find the property %1% in assembling
 
 //------------------ Outputs -----------------------
 MissingGeneratorParameterInSettings = parameter %1% not found in settings file for generator %2%
+NoSVCInFlatStartingPointMode =        cannot use a svc in flat starting point mode
```

Comment:
I would propose "simulation of secondary voltage controls is not possible with 'FLAT' starting point mode"